packages/dx-grid-core/src/plugins/sorting-state/reducers.ts

```diff
@@ -9,10 +9,10 @@ export const changeColumnSorting: PureReducer<ColumnSortingState, ChangeSortingP
 
   let nextSorting: any[] = [];
   if (keepOther === true) {
-    nextSorting = sorting.slice();
+    nextSorting = slice(sorting);
   }
   if (Array.isArray(keepOther)) {
-    nextSorting = sorting.slice()
+    nextSorting = slice(sorting)
```

Comment:
`slice` is not required here (`filter` creates new list)

Reply:
There is `slice` in the next conditions